python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -97,7 +97,11 @@ def _sliceSimData(islice):
                 indices = self.opsimtree.query_ball_point((sx, sy, sz), self.rad)
 
             for key in self.slicePoints.keys():
-                if (np.size(self.slicePoints[key]) > 1):
+                if len(np.shape(self.slicePoints[key])) == 0:
+                    shape = 0
+                else:
+                    shape = np.shape(self.slicePoints[key])[0]
+                if (shape == self.nslice):
```

Comment:
This is pretty unreadable out of context. Can you add some comments indicating what is being tested for and why? 
shape is already overloaded with the slicer.shape attribute (unless that's disappeared now?) so that makes this more confusing. Is 'shape' the shape of the slicer or of the map information at each point?